src/main/java/org/jitsi_modified/impl/neomedia/rtp/sendsidebandwidthestimation/SendSideBandwidthEstimation.java

```diff
@@ -532,9 +546,12 @@ private synchronized void updateMinHistory(long now_ms)
     }
 
     /**
-     * {@inheritDoc}
+     * void SendSideBandwidthEstimation::UpdateReceiverEstimate
```

Comment:
we generally try to use `{@link Classname#methodName}` for docs like this so they're clickable